controllers/model/grafanaRoute.go

```diff
@@ -21,6 +21,10 @@ func GetPath(cr *v1alpha1.Grafana) string {
 		return "/"
 	}
 
+	if cr.Spec.Ingress.Termination == "passthrough" {
+		return ""
+	}
+
```

Comment:
```suggestion
	if cr.Spec.Ingress.Termination == v1.TLSTerminationPassthrough {
		return ""
	}

```

I think it should rather rely on the value that's provided by the `github.com/openshift/api/route/v1` library

Reply:
@weisdd Thank you for your review. I fixed it.